src/device/controller/inputconvert/keymap/keymap.cpp

```diff
@@ -545,5 +550,7 @@ bool KeyMap::checkForSteerWhell(const QJsonObject &node)
 
 bool KeyMap::checkForDrag(const QJsonObject &node)
 {
-    return checkItemString(node, "key") && checkItemPos(node, "startPos") && checkItemPos(node, "endPos");
+    return checkItemString(node, "key") && checkItemPos(node, "startPos") && checkItemPos(node, "endPos") &&
```

Comment:
startDelay&dragSpeed not necessary, no need to check

Reply:
done 👍